spec/fixtures/mock_cdn_repo_remote/CocoaPods-version.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
is it possible to use one of the existing fixture repos for this?

Reply:
I've briefly considered this. 
However, this is the mock of the *remote* end of the CDN source, and so cannot be in the `spec-repos` dir. Also, CDNSource wasn't designed to be backward-compatible with non-sharded repos.
Is there a problem with having a new fixture for a new source type?

Reply:
Also, I can reduce the number of files in it if necessary.

Reply:
Ok, I just tried adapting the existing fixtures to the new test suite. The sharding was not the issue but it didn't go well due to the podspecs not being JSON.

Currently, I have not added RB podspec support because it seems out of scope to me. The only truly huge spec repo is the main one, and it's JSON-only.

What is the main pain point of adding a new test fixture?